data/human/human missions.txt

```diff
@@ -4366,6 +4367,35 @@ mission "AA Spaceport Reminder Resetter"
 		"spaceport reminder month" = "month"
 		fail
 
+# Old names of the mission above.
+disable mission
+	"00 Spaceport Reminder Resetter"
+	"aa Spaceport Reminder Resetter"
+	"AA Spaceport Reminder Resetter"
+
+mission "00 Spaceport Reminder Resetter"
+
+mission "aa Spaceport Reminder Resetter"
+
+mission "AA Spaceport Reminder Resetter"
+
+mission "Spaceport Reminder Resetter Patch"
+	landing
+	invisible
+	to offer
+		or
+			has "00 Spaceport Reminder Resetter: offered"
+			has "aa Spaceport Reminder Resetter: offered"
+			has "aa Spaceport Reminder Resetter: offered"
+	on offer
+		clear "00 Spaceport Reminder Resetter: offered"
+		clear "00 Spaceport Reminder Resetter: failed"
+		clear "aa Spaceport Reminder Resetter: offered"
+		clear "aa Spaceport Reminder Resetter: failed"
+		clear "AA Spaceport Reminder Resetter: offered"
+		clear "AA Spaceport Reminder Resetter: failed"
+		fail
```

Comment:
I simply wouldn't bother with all this. The reminder setter's mission conditions aren't used anywhere.